Sources/FoundationExtensions/Numeric/Numeric+Extensions.swift

```diff
@@ -127,6 +127,35 @@ extension BinaryInteger {
                               current: Double(current),
                               constrainedToValidPercentage: constrainedToValidPercentage)
     }
+
+    /// Combined operation of `linearProgress` and `linearInterpolation`.
```

Comment:
nitpick-mode: there is a space before the ///

Reply:
Wow, full indentation got broken when copied and pasted from bitbucket.
Thanks!